assets/helpers/url.js

```diff
@@ -1,5 +1,27 @@
 // @flow
 
+// ----- Types ----- //
+
+export type Domain
+  = 'thegulocal.com'
+  | 'code.dev-theguardian.com'
+  | 'theguardian.com'
+  ;
+
+export type Env = 'DEV' | 'CODE' | 'PROD';
+
+
+// ----- Setup ----- //
+
+const DOMAINS: {
```

Comment:
Seems quite shout-y. Does JS have the convention of constants being capitalised?

Reply:
JS conventions are somewhat... fluid 🙂.

I try to do this when using globals to make it super-clear that they're global constants so that:

a) you know that they're constants, and can be found near the top of the file, rather than by hunting around in the local scope.
b) you know that they shouldn't be changed (because JavaScript constants aren't that constant, the entire structure of this object could be changed).

In the style of C preprocessor macros.

Reply:
I do miss C preprocessor macros.